cdc/server.go

```diff
@@ -263,6 +264,10 @@ func (s *Server) run(ctx context.Context) (err error) {
 		return sorter.RunWorkerPool(cctx)
 	})
 
+	wg.Go(func() error {
```

Comment:
maybe we can run worker pool in `init` function?

Reply:
yep both run it in `init` function and explicit call work, but imho the `init` function is a little dangerous in some scenarios, which may make initialization step unclear (such as init sequence, unexpected init when we import a package but don't want it to init, etc)
such as the discussion in this article: https://medium.com/higher-order-functions/golang-how-to-sensibly-use-init-function-8077a4ec15e5